schematic/store/synapse.py

```diff
@@ -100,23 +101,53 @@ def __init__(
 
         self._query_fileview()
 
+    def _purge_synapse_cache(self):
+        '''
+        Purge synapse cache if it exceeds 7 GB
+        '''
+        # define the root directory
+        # set maximum synapse cache allowed
+        # set maximum ephemeral stroage allowed on AWS
+        root_dir = "/var/www/.synapseCache/"
```

Comment:
You could set this as an optional key argument of this function which makes it more flexible.

```
def _purge_synapse_cache(self, root_dir="/var..."):
```